packages/create-sitecore-jss/src/templates/nextjs/scripts/config/plugins/fallback.ts

```diff
@@ -0,0 +1,19 @@
+import { ConfigPlugin, JssConfig } from '..';
+
+/**
+ * This config will set fallback values for properties that were left empty
+ * If neither env, nor other places had a proper value, this will ensure a fallback is set
+ */
+class ComputedPlugin implements ConfigPlugin {
```

Comment:
Looks like c/p from computed plugin was done :) Should be "fallback" in this file.